### PR TITLE
Add LIP "Introduce unlocking condition for incentivizing certificate generation"

### PR DESCRIPTION
**Pull request by janhack**

### CODE REVIEW

**Review by ricott1 — APPROVED** [review]

File: proposals/lip-introduce_unlocking_condition.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+ For a height `h` of round `r` as input, this function returns the height of the last block of round `r`. In particular, if `h` is the last height of a round, we have `h = lastHeightOfRound(h)`.
+
+```python
+lastHeightOfRound(h):
+    r = roundNumber(h)
+    return r * ROUND_LENGTH + genesisHeight
+```
+
+Here `roundNumber` is a function defined in the [LIP "Define state and state transitions of DPoS module"][dpos-module-lip] and `genesisHeight` denotes the height of the genesis block as stored in the [DPoS module][dpos-module-lip].
```
Comment (ricott1) [inline proposals/lip-introduce_unlocking_condition.md:83]: Consider linking directly to the function when updating the links (e.g., link https://research.lisk.com/t/define-state-and-state-transitions-of-dpos-module/320#roundnumber-63 for now)

**Review by MaximeGagnebin — APPROVED** [review]
LGTM

**Review by ManuGowda — APPROVED** [review]
except some of the missing links to LIPs, LGTM.

File: proposals/lip-introduce_unlocking_condition.md (hunk 1 above)
Comment (janhack) [reply proposals/lip-introduce_unlocking_condition.md]: Thanks. I updated the link to link directly to the specific function.

**Review by karmacoma — APPROVED** [review]

**Pull request merged**